src/app/products/products.hbs

```diff
@@ -0,0 +1,39 @@
+<div class="products-container">
+  <nav class="sidebar">
+    <ul class="navbar">
+      {{#each categories.categories}}
+        <li class="nav-item">
+          <a class="nav-item-link item-{{@index}}" href="#">
+            <span>{{name}}</span>
+          </a>
+        </li>
+      {{/each}}
+    </ul>
+  </nav>
+  <nav class="topbar">
+    <select class="navbar selected">
+      {{#each categories.categories}}
+        <option class="nav-item">
+          <span class="nav-item-link">{{name}}</span>
```

Comment:
Span inside Option??? it only accepts text.

Reply:
Updated.